configs/localization/ssn/README.md

```diff
@@ -37,12 +37,34 @@ You can use the following command to train a model.
 python tools/train.py ${CONFIG_FILE} [optional arguments]
 ```
 
-Example: train SSN model on thumos14 dataset.
+Example: train SSN on THUMOS14 dataset.
 
 ```shell
 python tools/train.py configs/localization/ssn/ssn_r50_450e_thumos14_rgb_train.py
 ```
 
+Example: train SSN on ActivityNet-1.3 dataset.
```

Comment:
Is there any performance number about SSN on ANet?

Reply:
Not yet, I just make sure that users can train SSN with this config.